File: data_out/two-dim/BNR_2_order/TEST/value_plot.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Mon Apr 23 12:53:01 2018

@author: leixin
"""

import numpy as np
rho  = np.loadtxt('./RHO.csv',  delimiter='\t')
u    = np.loadtxt('./U.csv',    delimiter='\t')
v    = np.loadtxt('./V.csv',    delimiter='\t')
p    = np.loadtxt('./P.csv',    delimiter='\t')
phi_a= np.loadtxt('./PHI_a.csv',delimiter='\t')
z_a  = np.loadtxt('./Z_a.csv',  delimiter='\t')
num_x=len(rho[0])
num_y=len(rho)
if not (('d_x' in dir()) and ('d_y' in dir())):
    d_x=0.01;d_y=0.01
X = np.arange(-0.5*num_x, 0.5*num_x, 1)*d_x
Y = np.arange(-0.5*num_y, 0.5*num_y, 1)*d_y
X, Y = np.meshgrid(X, Y)
import matplotlib.pyplot as plt
from mpl_toolkits.mplot3d import Axes3D
fig = plt.figure()  
ax = Axes3D(fig)
'''
plot density 
'''
ax.plot_surface(X, Y, rho, cmap=plt.cm.winter)
plt.savefig("rho.png")
#plt.show()

File: data_in/two-dim/TEST/value_plot.m
rho  = load('./RHO.csv');
u    = load('./U.csv');
v    = load('./V.csv');
p    = load('./P.csv');
phi_a= load('./PHI_a.csv');
z_a  = load('./Z_a.csv');
phi_b= load('./PHI_b.csv');
z_b  = load('./Z_b.csv');
d_x=1;
d_y=1;
num_x=size(rho,2);
num_y=size(rho,1);
X = (1:num_x)*d_x;
Y = (1:num_y)*d_y;
[X, Y] = meshgrid(X, Y);
figure(1);
surf(X, Y, rho(num_x:-1:1,:));
title('rho');
figure(2);
surf(X, Y, p(num_x:-1:1,:));
title('p');
figure(3);
surf(X, Y, u(num_x:-1:1,:))
title('u')
figure(4)
surf(X, Y, v(num_x:-1:1,:));
title('v');
figure(5);
surf(X, Y, phi_a(num_x:-1:1,:));
title('phi_a');
figure(6);
surf(X, Y, phi_b(num_x:-1:1,:));
title('phi_b');
figure(7);
surf(X, Y, z_a(num_x:-1:1,:));
title('z_a');
figure(8);
surf(X, Y, z_b(num_x:-1:1,:));
title('z_b');
